### DISCUSSION TITLE
YOTO Daily / Radio

### DISCUSSION THREAD

**Discussion by TheManoeuvre** [cdnninja/yoto_ha#98] [Ideas]
Is it possible to start todays yoto daily or current radio feed based using a service call?

**Comment by cdnninja** [comment]
If you click into the media player entity click browse media.  For me yoto radio shows up.  Let me know if it is missing something you are looking for.  

Once it is playing you can check in the developer menu what the media id is and pass that via the service call.

**Reply by harrolf** [reply]
I don't see Yoto radio in the list. Only the cards that we have registered.

**Comment by TheManoeuvre** [comment]
I’m just updating this as I revisited the idea. It seems playing from your library does have a code but I couldn’t decipher how to detect the date. When playing Yoto daily from a button press on the Yoto however it was more plainly written.

Use  the below as the media content ID to get the current days episode

`3nC80-daily-{{now().strftime('%Y%m%d')}}`
